lighting/envMap.glsl

```diff
@@ -21,16 +22,20 @@ license:
 
 #ifndef SAMPLE_CUBE_FNC
 #if __VERSION__ >= 300
-#define SAMPLE_CUBE_FNC(CUBEMAP, NORM, LOD) texture(CUBEMAP, NORM, LOD)
+#define SAMPLE_CUBE_FNC(CUBEMAP, NORM, LOD) textureLod(CUBEMAP, NORM, LOD)
 #else
-#define SAMPLE_CUBE_FNC(CUBEMAP, NORM, LOD) textureCube(CUBEMAP, NORM, LOD)
+#define SAMPLE_CUBE_FNC(CUBEMAP, NORM, LOD) textureCubeLod(CUBEMAP, NORM, LOD)
```

Comment:
Noticing that GLSL 120 don't support textureCubeLod(CUBEMAP, NORM, LOD) I'm reverting this one to textureCube(CUBEMAP, NORM, LOD).

Reply:
@patriciogonzalezvivo These functions are not equivalent and do different things. `textureCube` is incorrect in that case.

Reply:
@patriciogonzalezvivo is it an option to use this extension? https://developer.mozilla.org/en-US/docs/Web/API/EXT_shader_texture_lod

Reply:
Apparently requires version 300.

"This extension is only available to WebGL1 contexts. In WebGL2, the functionality of this extension is available on the WebGL2 context by default. It requires GLSL #version 300 es."

Should we add a sampleCubeLod.glsl that gracefully replaces this under 300?

Reply:
Sure. I can do a bit more digging later.